src/corelibs/U2View/src/ov_msa/ColorSchemaDialogController.cpp

```diff
@@ -340,6 +340,7 @@ ColorSchemaSettingsPageWidget::ColorSchemaSettingsPageWidget(ColorSchemaSettings
     connect(addSchemaButton, SIGNAL(clicked()), SLOT(sl_onAddColorSchema()));
     connect(deleteSchemaButton, SIGNAL(clicked()), SLOT(sl_onDeleteColorSchema()));
     connect(colorSchemas, SIGNAL(currentRowChanged(int)), SLOT(sl_schemaChanged(int)));
+    connect(colorsDirEdit, SIGNAL(editingFinished()), SLOT(sl_validateSchemesDir()));
```

Comment:
This is not needed anymore because colorsDir is validated when OK button is pressed.

More than that this won't give you the desired effect: colorEditValue still stays incorrect after the message is shown to user.